docs/pages/clients/getting-started.md

```diff
@@ -32,28 +32,78 @@ The Development Client uses deep links to open projects from the QR code. If you
 
 </Tabs>
 
-## Building your Development Client
+## Building and installing your first custom client
 
-You can now build your project and launch it in your iOS simulator
+### In the cloud
+
+However you choose to manage your native projects, we recommend using [EAS Build](eas-build.md) for the smoothest experience, especially if you do not have experience with Xcode and gradle builds.
```

Comment:
ooh thank you! I thought gradle was the build system and Android Studio was more an IDE that made it easy to develop an Android project, but wasn't really in control of the build process.

Reply:
Technically you're actually right, but I think people (especially people who aren't so familiar with native tooling, that this sentence is intended for) think of them synonymously and are more likely to understand immediately what we're talking about if we say "Android Studio" instead of "Gradle".